src/audio/tensorflow/README.md

```diff
@@ -0,0 +1,34 @@
+# SOF Tensorflow Micro Integration
+
+The TFLM module integrates tensorflow micro into SOF as an audio classifiction and in the future an audio processing module. This module is currently still work in progress, and needs a few features to be completed before its ready for production.
+
+## Building
+
+Please run ```./scripts/tensorflow-clone.sh``` before building as this will clone all dependencies required to build tensor flow micro with optimized xtensa kernels.
+
+Please select the following in Kconfig
+```
+CONFIG_CPP=y
+CONFIG_STD_CPP17=y
+CONFIG_SOF_STAGING=y
```

Comment:
This could be an overlay alike sof/app/shell_overlay.conf or sof/app/perf_overlay.conf.